apps/src/applab/designElements/elementUtils.js

```diff
@@ -120,4 +120,13 @@ module.exports.isIdAvailable = function(newId, originalId) {
   }
 
   return true;
+};
+
+module.exports.isContentEditable = function (element) {
+  return element && element.tagName.toUpperCase() === 'DIV' && element.contentEditable;
```

Comment:
I see from https://github.com/code-dot-org/code-dot-org/pull/5491 you are checking the existence of any value. If that's working, them I'm fine with the `element.contentEditable !== ""` approach. I'll investigate a little further and then update the PR.